styles/aboutUs.scss

```diff
@@ -3,81 +3,123 @@
   width: 100%;
   width: 100vw;
 
-  padding-top: 100px;
-  margin-bottom: 100px;
+  padding-top: $spc-l;
+  padding-bottom: $spc-xl;
 
   flex-flow: column nowrap;
   align-items: center;
   text-align: center;
-  color: $font-color-dark;
-
-  background-image: url(assets/images/astronaut.svg);
-  background-size: 300px 300px;
-  background-position: 100% 20%;
-  background-repeat: no-repeat;
+  color: $font-color;
 
   .about-us-header {
     display: flex;
-    width: 100%;
-    width: 100vw;
-    margin-bottom: 60px;
-    flex-flow: column nowrap;
+    width: 75px;
```

Comment:
You can replace this with `rem`

Reply:
As you can see in line under, I used 75vw for the same property. I will replace px with % in that example.

Reply:
:see-no-evil:

Reply:
🙈